neo/sys/posix/posix_main.cpp

```diff
@@ -61,6 +61,17 @@ If you have questions concerning this license or the applicable additional terms
 #define					MAX_OSPATH 256
 #define					COMMAND_HISTORY 64
 
+
+namespace
+{
+	template<typename T>
+	inline T ignore_result( T x )
```

Comment:
In which header might I put such compiler instrumentation functions (or macros) optimally?